frontend/src/components/pages/RegistrantExperience/Waiver/index.tsx

```diff
@@ -1,13 +1,175 @@
 import React from "react";
-import { Box, Text } from "@chakra-ui/react";
+import {
+  Box,
+  Text,
+  VStack,
+  Checkbox,
+  ListItem,
+  OrderedList,
+  Stack,
+  Radio,
+  RadioGroup,
+  FormControl,
+  FormLabel,
+  Input,
+  Divider,
+  HStack,
+} from "@chakra-ui/react";
+import {
+  OptionalClauseResponse,
+  RequiredClauseResponse,
+  WaiverActions,
+  WaiverInterface,
+  WaiverReducerDispatch,
+} from "./waiverTypes";
 
-const Waiver = (): React.ReactElement => {
+interface WaiverPageProps {
+  waiverInterface: WaiverInterface;
+  waiverDispatch: React.Dispatch<WaiverReducerDispatch>;
+}
+
+const WaiverPage = ({
+  waiverInterface,
+  waiverDispatch,
+}: WaiverPageProps): React.ReactElement => {
   return (
     <Box>
-      <Text textStyle="displayXLarge">Camp Registration</Text>
-      <Text>Waiver</Text>
+      <VStack spacing={3} pt="1" align="stretch">
+        <Text textStyle="displayXLarge">
+          {waiverInterface.campName} Registration
+        </Text>
+        <Text textStyle="displayLarge" color="primary.green.100">
+          Waiver
+        </Text>
+        <Text textStyle="buttonSemiBold" pt="6">
+          In consideration of the participation of my child/children, (the
+          “child or children“), in the Focus on Nature photography camp/workshop
+          and all activities associated therewith, I, the undersigned
+          parent/guardian of the child/children, agree to the following terms
+          and conditions:
+        </Text>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text color="primary.green.100" textStyle="heading">
+          Identity and Waiver of Liability
+        </Text>
+
+        <OrderedList pl="10" spacing={5}>
+          {waiverInterface.requiredClauses.map(
+            (clause: RequiredClauseResponse, index: number) => (
+              <ListItem textStyle="bodyRegular" key={index}>
+                {clause.text}
+              </ListItem>
+            ),
+          )}
+        </OrderedList>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text textStyle="heading">
+          By clicking the checkbox below, you agree to the terms and conditions
+          above.
+        </Text>
+        <Checkbox
+          isChecked={waiverInterface.agreedRequiredClauses}
+          onChange={() =>
+            waiverDispatch({ type: WaiverActions.ClICK_REQUIRED_CLAUSE })
+          }
+        >
+          <Text textStyle="bodyRegular">
+            I agree to the above identity and liability waiver conditions{" "}
+            <Text as="span" color="text.critical.100" verticalAlign="super">
+              *
+            </Text>
+          </Text>
+        </Checkbox>
+      </VStack>
+      <Divider pt="4" pb="4" borderColor="border.secondary.100" />
```

Comment:
nit: `pt` and `pb` can be combined into `py`

Reply:
yep!